test/unit/module-register/input.js

```diff
@@ -0,0 +1 @@
+require('./register.mjs')
```

Comment:
Should this be `import('./register.mjs')` instead since this is presumably cjs?

Ideally you should be able to run `node test/unit/module-register/input.js` and it should work.

Reply:
Ah, I just looked at some of the other tests and assumed `input.js` was always CJS so did this to load an ESM test file

Reply:
Yes, this code in `register.mjs` cannot be in `input.js` because that is expected to be CJS. We need ESM to test `import.meta.url`.

We need this code to jump into ESM mode
```ts
require('./register.mjs')
```